using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +148,60 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+By default, a continuous aggregate is populated with data when it is created. 
+means that it is immediately populated with data. Since this can be a
+lengthy process it is sometimes desirable to defer materialization to
+a manual refresh or a policy that does it in the background. You can
+disable the immediate materialization of data using the `WITH NO DATA`
+option to `CREATE MATERIALIZED VIEW`.
+
+#### Automatic refreshing with a continuous aggregate policy
+
+You can refresh the continuous aggregate view manually as above, but
+it is also possible to automate the refreshing by adding a continuous aggregate 
+policy with [`add_continuous_aggregate_policy`][add-continuous-aggregate-policy]. 
+For example:
+
+```sql
+SELECT add_continuous_aggregate_policy('device_summary',
```

Comment:
The parameter choices here seem picked at random. I suggest we start by explaining the use case, i.e., what we (or the users) are trying to achieve. We need to setup a context where the reader understands that these parameter choices make sense. For instance, setting a `NULL` start interval will ensure that the aggregate will stay fully up-to-date with the source data from the beginning of time, including any deletions of source data. This is non-obvious as presented.

Therefore, I believe we need to focus on a couple of key use cases, as described in design docs. For instance:


1. The user wants the cagg to be always up-to-date with the source data, including following retention.
2. The user wants separate retention for caggs and source hypertable (keep cagg for longer, while dropping source data)
3. The user wishes to see recent aggregates immediately and either does not care about historical data or can aggregate it later (e.g., manually)

I think we should formulate our examples around such use cases and be clear with what we're trying to achieve with a particular policy.

I do realize that refocusing on use cases is a big change to this section, so I am happy to merge what we have and then rewrite again, if that's preferred.

Reply:
The last use case (3) is commonly requested among users. For instance here: https://github.com/timescale/timescaledb/issues/1634#issuecomment-633861621

I think it would be a mistake to not include this somehow. But it need not happen as part of this PR. I imagine we'll continuous tweaking.